src/mon/MDSMonitor.cc

```diff
@@ -1815,8 +1815,7 @@ class SetHandler : public FileSystemCommandHandler
         return -EINVAL;
       }
 
-      if (!fs->mds_map.allows_multimds() && n > fs->mds_map.get_max_mds() &&
```

Comment:
The problem with this change is in the case where someone has:

1.  setallow_multimds tru
2.  set max_mds to something >1
3.  set allow_multimds false

In this situation, we should allow people to decrease max_mds for their filesystem.

So I think just drop this commit?